packages/contracts/deploy/helpers.ts

```diff
@@ -614,5 +650,27 @@ export function getManagingDAOMultisigAddress(
   return address;
 }
 
+export async function skipIfZkSync(
+  hre: HardhatRuntimeEnvironment,
+  stage: string
+) {
+  if (ZK_SYNC_NETWORKS.includes(hre.network.name)) {
+    console.log(`Skipping deployment stage: ${stage} due to zkSync network`);
+    return true;
+  }
+  return false;
+}
+
+export async function skipIfNotZkSync(
+  hre: HardhatRuntimeEnvironment,
+  stage: string
+) {
+  if (!ZK_SYNC_NETWORKS.includes(hre.network.name)) {
+    console.log(`Skipping deployment stage: ${stage} due to zkSync network`);
```

Comment:
```suggestion
    console.log(`Skipping deployment stage: ${stage} due is not zkSync network`);
```\

Reply:
I don't think the suggestion is fully grammatically correct.

Reply:
yeap, sorry, updated